### DISCUSSION TITLE
plugins/browser/

### DISCUSSION THREAD

**Discussion by giscus** [cargo-youth/ToolDiy#58] [Announcements]
# plugins/browser/

https://tooldiy.ry.rs/plugins/browser/

**Comment by Mc-GrowlR** [comment]
我在火狐浏览器上常用的插件：
1. MarkDownload - Markdown Web Clipper：可以将网页保存为 md 文档，我最常用的功能是直接复制网页的带有网页标题的 `md` 样式的链接。示例：[Browser - ToolDiy](https://tooldiy.ry.rs/plugins/browser/)
2. Tab Center Reborn：可以为火狐浏览器提供侧边标签栏功能。安装此插件后，你可以将顶部标题栏和地址栏给隐藏，到达真正的简约风格。当然，二者可以只隐藏其中一个。
3. Tab Session Manager： 保存、还原窗口与标签页的状态，支持自动保存。类似于 `Edge` 浏览器的集锦功能。
4. 沙拉查词-聚合词典划词翻译。如名字，鼠标选取文本之后可以将其翻译成汉语。
5. Plasma Integration：使用KDE桌面的必备插件，搜索可以检索火狐浏览器的标签页和历史记录。
